composer.json

```diff
@@ -49,14 +50,18 @@
         "sentry/sentry": "^1.0",
         "segmentio/analytics-php": "^1.8",
         "ramsey/uuid": "^3.9",
-        "prestashopcorp/oauth2-prestashop": "^2.0",
+        "prestashopcorp/oauth2-prestashop": "dev-feat/token-validator",
```

Comment:
is there any official version to be released? branching to a dev branch is ok for testing but not in production